docs/3.0/customization.md

```diff
@@ -147,145 +147,6 @@ It's recommended you don't store your current user here but using the [`current_
 
 You can access the context data with `::Avo::App.context` object.
 
-## Eject
```

Comment:
Let's leave behind a link to the new page and also the `## Eject` headline so whoever is using the `/customization.html#eject` anchor sees this.